### DISCUSSION TITLE
Unable to access localhost:3000 for Next.js app on Windows - Connection Refused

### DISCUSSION THREAD

**Discussion by erbill** [vercel/next.js#48126] [Help] [answered]
### Summary

I'm having trouble running a Next.js app on my Windows machine. I've followed the standard setup process, but when I try to access the app at localhost:3000 or any other port, I get a "Connection Refused" error. I've tried multiple browsers, but the issue persists.

Here's what I've tried so far:

Creating a new Next.js app using npx create-next-app.
Running the app using npm run dev.
Checking my hosts file to ensure the correct localhost mapping.
Temporarily disabling my firewall and antivirus.
Restarting my computer.
Trying a different browser.
Creating a simple test.html file and attempting to access it directly.
Despite these efforts, I still can't access the app on localhost. I'm not sure what else to try, and I'm looking for any suggestions or guidance on how to resolve this issue.

Any help would be greatly appreciated. Thank you!

### Additional information

```js
My system information:

Windows version: (Windows 11)
Node.js version: (Node.js v18.15.0)
Next.js version: (Next.js 13.3.0)
```

### Example

even a sample app doesn't work.

**Comment by samthatcode** [comment]
I have the same issues with my project. still looking for answers. Soon as i get one that works, I will make another reply here on how was i able to get it to work. If you get answers too, kindly  do tell

**Reply by erbill** [reply]
Yes it's not working still for me. I've deployed with Vercel without working on local, It's not the best way to get around but at least I keep improving my project while trying to figure out why I can't run it on my local. 

Also today I've run some other python code on local server, so it might not be about windows firewall afterall

**Reply by samthatcode** [reply, marked as answer]
Hey man. I was just about to come on here and reply back to my response earlier. I got it working on my Local machine. I had to install node 18 and restart my machine and it's working fine now. You should probably do the same. I hope it works for you.


Thanks for the feedback

**Comment by w1am** [comment]
For me, the reason the dev server stopped working was that something in my cookies was causing errors. All I had to do was delete all the cookies, and it worked again.

**Comment by gazedash** [comment]
Not working

**Reply by Bashek** [reply]
Try to use `0.0.0.0` instead of `localhost`

**Reply by cglacet** [reply]
`localhost` works on mac, but doesn't on linux, on the other hand `0.0.0.0` worked.

**Comment by ukpagrace** [comment]
What worked for me, was killing all the running processes with the port, in my case 3000
